### PR TITLE
3066 em api update verbiage for udp per eaton policy

### PR DESCRIPTION
**Pull request by berkererdinie**

### CODE REVIEW

**Comment by NasibMansour** [comment]
It's a little weird now that we updated everything but the repo name is emcb-udb-master, what do you think @Moneil97? should we push to a new repo or something?

**Comment by Moneil97** [comment]
> It's a little weird now that we updated everything but the repo name is emcb-udb-master, what do you think @Moneil97? should we push to a new repo or something?

Github does allow updating of the repo name. But there are 2 issues with that:
1. I don't seem to have the permissions to do that. Looks like you and Jason are marked as "owner" so you probably can do it.
2. Will the old links redirect to the new name? We would want to update the link in any of our documentation anyway, but it would be nice not to break the old link for anyone that has it saved.

**Comment by Moneil97** [comment]
From https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository 
```
When you rename a repository, all existing information, with the exception of project site URLs, is automatically redirected to the new name, including:

Issues
Wikis
Stars
Followers
```

So I think we should be ok to rename the repo.

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 name: Question
-about: The issue tracker is not for questions. Please ask questions on https://stackoverflow.com/questions/tagged/emcb-udp-master or contact Eaton.
+about: The issue tracker is not for questions. Please ask questions on https://stackoverflow.com/questions/tagged/emcb-udp-coordinator or contact Eaton.
 
 ---
 
 🚨 The issue tracker is not for questions 🚨
 
-If you have a question, please ask it on https://stackoverflow.com/questions/tagged/emcb-udp-master or contact Eaton.
+If you have a question, please ask it on https://stackoverflow.com/questions/tagged/emcb-udp-coordinator or contact Eaton.
```
Comment (Moneil97) [inline .github/ISSUE_TEMPLATE/question.md:9]: Do we want to keep this stackoverflow link? Feels like we should just use the issues section of this repo? Or do we have a better place? The tag has never been used and nobody is monitoring it anyway.

File: README.md [hunk 2]
```diff
@@ -46,25 +46,25 @@ Terminology:
   EM Coodinator)
 
 **NOTE:** Some documentation and much of the implementation in this SDK utilize
-Master/Slave terminology, but will be updated to use Coordinator/Node terminology
+Coordinator/Node terminology, but will be updated to use Coordinator/Node terminology
```
Comment (Moneil97) [inline README.md]: We can now remove this note

File: README.md [hunk 3]
```diff
@@ -161,9 +161,9 @@ $ npm install npm -g
...
-[dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-master
-[dev-dependency-status-svg]: https://david-dm.org/EatonEM/emcb-udp-master/dev-status.svg
-[dev-dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-master#info=devDependencies
-[npm-version-svg]: https://badge.fury.io/js/emcb-udp-master.svg
-[npm-version-link]: https://badge.fury.io/js/emcb-udp-master
\ No newline at end of file
+[dependency-status-svg]: https://david-dm.org/EatonEM/emcb-udp-coordinator.svg
+[dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-coordinator
+[dev-dependency-status-svg]: https://david-dm.org/EatonEM/emcb-udp-coordinator/dev-status.svg
+[dev-dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-coordinator#info=devDependencies
```
Comment (Moneil97) [inline README.md]: I don't know what these `https://david-dm.org/` links are but it seems the site no longer exists so I guess we remove these?

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (berkererdinie) [reply .github/ISSUE_TEMPLATE/question.md:9]: @NasibMansour should it be kept or removed?

Comment (NasibMansour) [reply .github/ISSUE_TEMPLATE/question.md:9]: I agree with Michael, let's remove it

**Pull request merged**